dashboard/test/controllers/api/v1/schools_controller_test.rb

```diff
@@ -0,0 +1,83 @@
+require 'test_helper'
+
+class Api::V1::SchoolsControllerTest < ActionController::TestCase
+  self.use_transactional_test_case = true
```

Comment:
pretty sure this line isn't necessary unless a `setup_all` block is used (though harmless to leave in, since the tests are already passing).

Reply:
I can remove it. I copied it from another test.